packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.test.ts

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {act} from '@testing-library/react'
+import nock from 'nock'
+import {DEFAULT_TEST_HOST, renderHookWithProviders} from '../../test-utils'
+import {getQueryKeysMatrix, ShopperBasketMutationType, useShopperBasketsMutation} from './mutation'
+import {useBasket} from './query'
+import {useCustomerBaskets} from '../ShopperCustomers/query'
+
+const CUSTOMER_ID = 'CUSTOMER_ID'
+const BASKET_ID = 'BASKET_ID'
+
+jest.mock('../../auth/index.ts', () => {
+    return jest.fn().mockImplementation(() => ({
+        ready: jest.fn().mockResolvedValue({access_token: '123'})
+    }))
+})
+
+jest.mock('../useCustomerId.ts', () => {
+    return jest.fn().mockReturnValue(CUSTOMER_ID)
+})
+
+// TODO: should this be in test-utils? So it can be reused across test suites
+const mockMutationEndpoints = (matchingPath: string, options?: {errorResponse: number}) => {
```

Comment:
The new test refactor requires us to nock all the endpoints.
 I think, yes, we should move this function to test-utils.tsx, so we can re-use it and be consistent in the rest of the tests.
 
I'd suggest also implementing a similar generic function in the test-utils.tsx to nock the query endpoints.
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/a4d8dd4190e9b2254402f7331a4c1252962fe1df/packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.test.ts#L74-L91